DVS.psm1

```diff
@@ -429,10 +433,10 @@ Function Enum-HostList {
             [Array]::Reverse($IPAddress)
             $IPAddress = ([System.Net.IPAddress]($IPAddress)).IPAddressToString
             if(Find-InArray -Content $IPAddress -Array $IPAddressList) {
-                %{"127.0.0.1"}
+                ForEach-Object{"127.0.0.1"}
                 continue
             }
-            %{$IPAddress}
+            ForEach-Object{$IPAddress}

```

Comment:
Using with Foreach-object is too slow, please change it as the original script

Reply:
What do you mean with "too slow"? `%` is an alias for `Foreach-Object` and this should therefore not change anything in performance. https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-alias?view=powershell-7